src/InputPicker/InputPicker.tsx

```diff
@@ -594,10 +594,6 @@ const InputPicker: PickerComponent<InputPickerProps> = React.forwardRef(
       }
     }, [readOnly]);
 
-    const handleBlur = useCallback(() => {
-      setOpen(false);
```

Comment:
Calling set Open(false) in onBlur is incorrect because Dropdown onBlur is also called when clicking on Dropdown items. Closing Dropdown works without this function, when clicking outside the component.